packages/core/database/lib/dialects/sqlite/index.js

```diff
@@ -24,6 +24,10 @@ class SqliteDialect extends Dialect {
     fse.ensureDirSync(dbDir);
   }
 
+  useReturning() {
```

Comment:
Afaik there was a problem with this between knex and sqlite and it was not working. it's mentioned several times in knex that they don't have a common "returning" system and the returned data won't be the same between DBs

Have you looked at the different return format from pg / MySQL & sqlite ?

Reply:
🤔 I'll do some testing